TKeazirian.HTTPServer/Handler/SimpleOptionsHandler.cs

```diff
@@ -0,0 +1,46 @@
+namespace TKeazirian.HTTPServer.Handler;
+
+using Response;
+using Request;
+
+public class SimpleOptionsHandler : Handler
+{
+    public override List<string> AllowedHttpMethods()
+    {
+        return new List<string>() { "GET" };
+    }
+
+    public override Response HandleResponse(Request request)
+    {
+        if (request.GetRequestPath() == "/method_options")
+        {
+            return new ResponseBuilder()
+                .SetStatusCode(HttpStatusCode.Ok)
+                .SetHeaders("Allow", AddHeadToAllowedMethods())
+                .Build();
+        }
+
+        return new ResponseBuilder()
+            .SetStatusCode(HttpStatusCode.Ok)
+            .SetHeaders("Allow", AddHeadPutPostToAllowedMethods())
+            .Build();
+    }
+
+    private string AddHeadToAllowedMethods()
+    {
+        List<string> allowHead = new List<string>() { "HEAD, OPTIONS" };
+        allowHead.AddRange(AllowedHttpMethods());
+
+        string allowedMethods = string.Join(", ", allowHead);
+        return allowedMethods;
+    }
+
+    private string AddHeadPutPostToAllowedMethods()
+    {
+        List<string> allowHead = new List<string>() { "HEAD", "OPTIONS", "PUT", "POST" };
```

Comment:
If you are supporting PUT and POST, I would expect to see something building those responses in this Handler.

Reply:
That's the confusing part.  With the acceptance tests, they only test that they are allowed but not that they do anything with them.  Another layer!

Reply:
time to build the some of the structure? let the tests guide you.

Reply:
I have updated it - not sure if it's in the right direction but maybe?!